spec/fixtures/files/datahashes/core/media1.json

```diff
@@ -27,10 +28,10 @@
     "copyrightStatement": "For the ones we have",
     "type": "document;moving_image;still_image;sound",
     "coverage": "Area Code",
-    "dateGroup": null,
+    "dateGroup": "8/15/2020",
```

Comment:
Looks like you were maybe running into the same issue Erick was running into [here](https://github.com/collectionspace/collectionspace-mapper/pull/28#pullrequestreview-484191424). Since we don't seem to be able to remove a date from record and reset `scalarValuesComputed` to false, I just added a date value.